app/views/pages/developer-guide/apis/creating-items-from-external-api-call.liquid

```diff
@@ -21,11 +21,23 @@ Creating items from an external API call is a four-step process:
 
 <div data-autosteps></div>
 
-### Step 1: Create Model Schema
+### Step 1: Prepare your endpoints' headers
 
-Create the following Model Schema:
+Your endpoints will serve content as JSON. You need to create a Liquid file under `app/views/partials/shared/api/headers.liquid` to render the necessary headers:
 
-##### app/model_schemas/flowers.yml
+```liquid
+{% raw %}
+{
```

Comment:
Would be nice to add the whole thing with `---` and `response_headers` for context